lib/fxpay/settings.js

```diff
@@ -10,6 +10,9 @@
     //
     // Disallow receipts belonging to other apps.
     allowAnyAppReceipt: false,
```

Comment:
I saw someone (not me) make a case (where? I can't remember) about building app A and selling app B as some kind of premium but then wanting to share receipts between the two for some reason. I never fully understood the use case though.

Do you think we should remove it and just wait until someone asks for it?